docs/core/tools/global-json.md

```diff
@@ -65,15 +140,30 @@ dotnet new globaljson --sdk-version 2.2.100
 > The matching rules are governed by the apphost, which is part of the .NET Core runtime.
 > The latest version of the host is used when you have multiple runtimes installed side-by-side.
 
-Starting with .NET Core 2.0, the following rules apply when determining which version of the SDK to use:
+## [.NET Core 3.x](#tab/netcore3x)
+
+Starting with .NET Core 3.0, the following rules apply when determining which version of the SDK to use:
 
-- If no *global.json* file is found or *global.json* doesn't specify an SDK version, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
+- If no *global.json* file is found, or *global.json* doesn't specify an SDK version nor an `allowPrerelease` value, the latest installed SDK version is used (equivalent to setting `rollForward` to `latestMajor`). Whether the latest SDK version can be release or prerelease depends on how `dotnet` is being invoked.
+  - If you're **not** in Visual Studio, prerelease versions are considered.
+  - If you are in Visual Studio, it uses the prerelease status requested. That is, if you're using a Preview version of Visual Studio or you set the **Use previews of the .NET Core SDK** option (under **Tools** > **Options** > **Environment** > **Preview Features**), prerelease versions are considered; otherwise, only release versions are considered.
+- If a *global.json* file is found that doesn't specify an SDK version but it specifies an `allowPrerelease` value, the latest installed SDK version is used (equivalent to setting `rollForward` to `latestMajor`). Whether the latest SDK version can be release or prerelease depends on the value of `allowPrerelease`. `true` indicates prerelease versions are considered; `false` indicates that only release versions are considered.
+- If a *global.json* file is found and it specifies an SDK version:
+
+  - If no `rollFoward` value is set, it uses `major` as the default `rollForward` policy. Otherwise, check each value and their behavior in the [rollForward](#rollforward) section.
+  - Whether prerelease versions are considered and what's the default behavior when `allowPrerelease` isn't set is described in the [allowPrerelease](#allowprerelease) section.
+
+## [.NET Core 2.x](#tab/netcore2x)
+
```

Comment:
@mairaw Add your magic formatting here

> The matching rules are governed by the apphost, which is installed as part of the .NET Core runtime.
> These rules are used unless a 3.0 or higher version of the SDK or Runtime have ever been installed, since the latest version of the host is used when you have multiple runtimes installed side-by-side.

Reply:
We already have a similar warning before the tabs:
https://review.docs.microsoft.com/en-us/dotnet/core/tools/global-json?branch=pr-en-us-15382&tabs=netcore3x#matching-rules

Not sure what you'd like to do here

Reply:
NM